tests/integration/test_charm.py

```diff
@@ -28,6 +31,7 @@
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 CHARM_NAME = METADATA["name"]
+CONTAINERS_UID_MAP = generate_container_securitycontext_map(METADATA)
```

Comment:
suggestion (non-blocking): do we want to use the more intuitive constant name from [here](https://github.com/canonical/pvcviewer-operator/pull/116/files#diff-08291d7f4ad7b2caecacb6ae26af540ebbaede9fe42d5e8e1b39d2bdefe4c2d4)? up to you